x/wasm/client/cli/gov_tx.go

```diff
@@ -23,27 +24,44 @@ func ProposalStoreCodeCmd() *cobra.Command {
 				return err
 			}
 
-			src, err := parseStoreCodeArgs(args, clientCtx)
+			src, err := parseStoreCodeArgs(args[0], clientCtx.FromAddress, cmd.Flags())
```

Comment:
This change is due to the genesis changes?
Or a more general update/refactor to the new stargate proposals?

Looks fine, just trying to understand why this is here

Reply:
Ja, I am also using this parseStoreCodeArgs for the genesis message command